js/spotify.js

```diff
@@ -1,12 +1,43 @@
-// API Docs at: 
+// API Docs at:
 // https://developer.spotify.com/web-api/search-item/
+$(document).ready(function () {
+  $('form').on('submit', function (e) {
+    e.preventDefault();
+  var keyword = $('#search-keyword').val();
+  searchByArtist();
```

Comment:
Your searchByArtist function requires a parameter so you would have wanted to pass it something here when it is being invoked. Maybe `searchByArtist(keyword);`